agent/consul/client_serf.go

```diff
@@ -27,6 +28,7 @@ func (c *Client) setupSerf(conf *serf.Config, ch chan serf.Event, path string) (
 	conf.Tags["vsn_min"] = fmt.Sprintf("%d", ProtocolVersionMin)
 	conf.Tags["vsn_max"] = fmt.Sprintf("%d", ProtocolVersionMax)
 	conf.Tags["build"] = c.config.Build
+	conf.Tags["reconnect_timeout"] = c.config.AdvertiseReconnectTimeout.String()
```

Comment:
What is our serf tag budget like these days after we removed the big consumer over in ent? Should this key be more abbreviated like we do for feature flags?

Reply:
Renamed to `rc_tm`